readme.md

```diff
@@ -36,14 +36,21 @@ Install the `youtube-music-bin` package from the AUR. For AUR installation instr
 ## Available plugins:
 
 - **Ad Blocker**: block all ads and tracking out of the box
-- **Downloader**: download to MP3 directly from the interface (youtube-dl)
+- **Downloader**: downloads MP3 [directly from the interface](https://user-images.githubusercontent.com/61631665/129977677-83a7d067-c192-45e1-98ae-b5a4927393be.png) [(youtube-dl)](https://github.com/ytdl-org/youtube-dl)
 - **No Google Login**: remove Google login buttons and links from the interface
 - **Shortcuts**: use your usual shortcuts (media keys, Ctrl/CMD + F…) to control YouTube Music
 - **Navigation**: next/back navigation arrows directly integrated in the interface, like in your favorite browser
-- **Auto confirm when paused**: when the "Continue Watching?" modal appears, automatically click "Yes"
+- **Auto confirm when paused**: when the ["Continue Watching?"](https://user-images.githubusercontent.com/61631665/129977894-01c60740-7ec6-4bf0-9a2c-25da24491b0e.png) modal appears, automatically click "Yes"
 - **Hide video player**: no video in the interface when playing music
-- **Notifications**: display a notification when a song starts playing
+- **Notifications**: display a [notification](https://user-images.githubusercontent.com/78568641/114102651-63ce0e00-98d0-11eb-9dfe-c5a02bb54f9c.png) when a song starts playing
 - **Touchbar**: custom TouchBar layout for macOS
+- [**Discord**](https://discord.com/): show your friends what you listen to with [Rich Presence](https://user-images.githubusercontent.com/28219076/104362104-a7a0b980-5513-11eb-9744-bb89eabe0016.png)
+- **In-app menu**: allows you to [control app from your tray](https://user-images.githubusercontent.com/61631665/129975597-65491d42-47bd-426a-a1a0-7c91114255ff.png) and [gives bars a fancy, dark look](https://user-images.githubusercontent.com/78568641/112215894-923dbf00-8c29-11eb-95c3-3ce15db27eca.png)
+- [**Last.fm**](https://www.last.fm/): scrobbles support
+- **Playback speed**: listen fast, listen slow! [Adds a slider that controls song speed](https://user-images.githubusercontent.com/61631665/129976003-e55db5ba-bf42-448c-a059-26a009775e68.png)
+- **Precise volume**: customizable volume steps for more comfort, allows controlling the volume in precisely using mousewheel
```

Comment:
typo, should be "allows controlling the volume precisely using mousewheel"
(without *in*)